packages/admin/admin/src/common/buttons/feedback/FeedbackButton.tsx

```diff
@@ -36,9 +36,8 @@ export interface FeedbackButtonProps
             root: typeof LoadingButton;
             tooltip: typeof CometTooltip;
         }>,
-        LoadingButtonProps {
-    loading?: boolean;
-    hasErrors?: boolean;
+        Omit<LoadingButtonProps, "loading"> {
+    onClick: () => Promise<void>;
```

Comment:
@nsams @thomasdax98 what do you think about using the promise states (pending, resolved, rejected) to handle the display states of the button instead of `loading`, `hasErrors` etc. props? I think this is easier to understand and less error-prone as the state transitions are pretty clear. Could there be use cases where I wouldn't want to have this?